Makefile

```diff
@@ -592,21 +593,27 @@ scale-image: scale-build clean-image
 	cp bin/linux/profiler scale/image/bin/profiler
 	cp bin/linux/chaos scale/image/bin/chaos
 	chmod +w scale/image/bin/*
-	docker build -t stackrox/scale:$(TAG) -f scale/image/Dockerfile scale
-	docker tag stackrox/scale:$(TAG) quay.io/$(CI_QUAY_REPO)/scale:$(TAG)
+	docker build \
+		-t stackrox/scale:$(TAG) \
+		-t quay.io/rhacs-eng/scale:$(TAG) \
```

Comment:
Don't we need that image in `quay.io/statckrox-io` too?

Reply:
I don't know how these are used. I retained the original behavior that they were only tagged for `rhacs-eng`. I did not want to dig too much in the situation to see if we also need providing them for community users.

Reply:
The only place I found for `CI_QUAY_REPO` was in `Makefile`.  https://github.com/stackrox/stackrox/blob/86478ce96be24a5b58af4b75472f0621519033ae/Makefile#L36-L39 
PTAL at it in the entirety. `CI_QUAY_REPO` existed but was never set to anything different than `rhacs-eng`. I guess it is some unfinished idea.